src/libs/Network/NetworkStore.ts

```diff
@@ -98,7 +98,7 @@ function getAuthToken(): string | null | undefined {
 }
 
 function isSupportRequest(command: string): boolean {
-    return [WRITE_COMMANDS.OPEN_APP, SIDE_EFFECT_REQUEST_COMMANDS.RECONNECT_APP, SIDE_EFFECT_REQUEST_COMMANDS.OPEN_REPORT].some((cmd) => cmd === command);
+    return [WRITE_COMMANDS.OPEN_APP, SIDE_EFFECT_REQUEST_COMMANDS.RECONNECT_APP, SIDE_EFFECT_REQUEST_COMMANDS.OPEN_REPORT, READ_COMMANDS.SEARCH].some((cmd) => cmd === command);
```

Comment:
i know you mention no need to block on auth but i wanted to double check. Im not fully aware of this flow, if this App code goes to staging before the auth PR is live won't this feature throw server errors because the command doesn't support the support token yet? Or are we saying we're okay with those errors for sometime until the auth PR is live?

Reply:
Auth would return a `411 Insufficient privileges` error code, and then PHP would catch it [here](https://github.com/Expensify/Web-Expensify/blob/02da50b73ec2401c483307a97178956f65c5bd19/api.php#L4411-L4416) and reformat the response returned to the client. It's all logged as `info`s though, there are no alerts or warns, which is why I think it's fine if this PR gets deployed the Auth change. Nothing will be any more broken than it is now :D

Reply:
haha alright, thanks for clarifying.